### DISCUSSION TITLE
Can I change size for topbar?

### DISCUSSION THREAD

**Discussion by Rxnel-ysr** [end-4/dots-hyprland#1016] [Q&A] [answered]
This what I means
![image](https://github.com/user-attachments/assets/18587fad-ce42-4b6d-ab65-3099609c507b)
 
 I can do it before, but i forgot how😅
![Pasted image1](https://github.com/user-attachments/assets/2c51a0ac-5b42-46e3-ae7a-76baf16622fe)

I know it just slight different but it hurt my eyes

thank you

**Comment by kxrur** [comment]
it's all in the wiki [resize-ui](https://end-4.github.io/dots-hyprland-wiki/en/i-i/03config/#resize-the-ui--change-font-size)

**Comment by PharmaRacist** [comment, marked as answer]
at the _bar.scss look for .bar-height class and adjust it to ur liking

**Reply by UcraNerd** [reply]
doesn't change anything for me. The problem is that the only way to change the size is by changing the font size, but if it isn't 11pt then it looks ass

**Reply by PharmaRacist** [reply]
we moved to quick shell now its perfect go to ~/config/quickshell/modules/common/appearance in qtObject bar u will find bar height

**Reply by UcraNerd** [reply]
> we moved to quick shell now its perfect go to ~/config/quickshell/modules/common/appearance in qtObject bar u will find bar height
> 

Yoo, it looks sick! 🔥 I will try it as soon as I have time to install it.

**Comment by Rxnel-ysr** [comment]
oh I see, thank you guys
